src/ui/song-notes.jsx

```diff
@@ -48,6 +58,14 @@ SongNotes.propTypes = {
   children: PropTypes.element.isRequired,
   songCount: PropTypes.number.isRequired,
   songName: PropTypes.string.isRequired,
+  location: PropTypes.shape({
+    generalLocation: PropTypes.string.isRequired,
+    detailedLocation: PropTypes.string.isRequired,
+  }),
+};
+
+SongNotes.defaultProps = {
+  location: undefined,
```

Comment:
To be more consistent with existing code:
```suggestion
  location: null,
```